p2p/security/noise/transport_test.go

```diff
@@ -714,3 +715,11 @@ func TestHandshakeWithTransportEarlyData(t *testing.T) {
 		})
 	}
 }
+
+func BenchmarkNoise(b *testing.B) {
```

Comment:
Can we have them all in one place? `p2p/test/transport` or `p2p/test/security` would probably be a good place.

Reply:
I moved it to `p2p/test/security`